defund-private-2/README.md

```diff
@@ -0,0 +1,111 @@
+# Defund Private Testnet: defund-private-2
+
+### Quick Links
+Genesis: TBA
+
+Git tag: v0.1.0
+
+Block explorer: **coming soon**
+
+Seeds: TBA
+
+#### Hardware Requirements
+Here are the minimal hardware configs required for running a validator/sentry node
+ - 16GB RAM
+ - 4vCPUs
+ - 200GB Disk space
+
+#### Software Requirements
+- Ubuntu 20.04 or higher
+- [Go v1.19.1](https://golang.org/doc/install)
+
+### Installation Steps
+
+#### Install Prerequisites 
+
+The following are necessary to build defund from source. 
+
+##### 1. Basic Packages
+
+```sh
+# update the local package list and install any available upgrades 
+sudo apt-get update && sudo apt upgrade -y 
+# install toolchain and ensure accurate time synchronization 
+sudo apt-get install make build-essential gcc git jq chrony -y
+```
+
+
+##### 2. Install Go
+Follow the instructions [here](https://golang.org/doc/install) to install Go.
+
+Alternatively, for Ubuntu LTS, you can do:
+
+```sh
+wget https://golang.org/dl/go1.19.1.linux-amd64.tar.gz
+sudo tar -C /usr/local -xzf go1.19.1.linux-amd64.tar.gz
+```
+
+Unless you want to configure in a non standard way, then set these in the `.profile` in the user's home (i.e. `~/`) folder.
+
+```sh
+cat <<EOF >> ~/.profile
+export GOROOT=/usr/local/go
+export GOPATH=$HOME/go
+export GO111MODULE=on
+export PATH=$PATH:/usr/local/go/bin:$HOME/go/bin
+EOF
+source ~/.profile
+go version
+```
+
+Output should be: `go version go1.19.1 linux/amd64`
+
+
+#### Install Defund from source
+
+##### 1. Clone repository
+```sh
+git clone https://github.com/defund-labs/defund
+cd defund
+git checkout v0.1.0
+make install
+```
+
+#### 2. Init chain
+```sh
+defundd init $MONIKER_NAME --chain-id defund-private-2
```

Comment:
$MONIKER_NAME variable has not yet been set. change to `<moniker>` and add note to replace.